src/components/Incident.tsx

```diff
@@ -21,29 +23,56 @@ export type IncidentProps = {
   readonly?: boolean;
 } & React.HTMLProps<HTMLDivElement>;
 
+export type IncidentHighlightProps = {
+  incident: IncidentData;
+};
+
+export const IncidentHighlights: React.FC<IncidentHighlightProps> = ({
+  incident,
+}) => {
+  const { data: eventData } = useCurrentEvent();
+  const { data: game } = useRulesForEvent(eventData);
+
+  const firstRule = incident.rules[0];
+  const firstRuleIcon = game?.rulesLookup?.[firstRule]?.icon;
+
+  return (
+    <>
+      <span key={`${incident.id}-name`}>{incident.team}</span>
+      {"•"}
+      <span key={`${incident.id}-match`}>
+        {incident.match ? matchToString(incident.match) : "Non-Match"}
+      </span>
+      {"•"}
+      <span>
+        {incident.rules.length >= 2 ? (
```

Comment:
I think the `<SG6> + 1` style here is best in the situation where we have multiple rules

Reply:
See 0fe95c637c8d3ea71396e76ee593906f729b1ff2